.travis.yml

```diff
@@ -14,6 +16,8 @@ install:
   - docker-compose run web bash -c "rake db:setup"
   - docker-compose run web bash -c "rake db:migrate"
   - docker-compose run web bower install --allow-root
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then gem install danger; fi'
```

Comment:
Well as per [Travis documentation](https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables) the variable sets to "false" if it isn't a pull request. If it's a pull request it gives the pull requests number not a Boolean value. Not sure if any non-zero will be treated as true.